cluster/cluster.go

```diff
@@ -5,6 +5,12 @@ import (
 	"net/url"
 )
 
+const (
+	OSD = "OSD"
```

Comment:
you could even create a `ClusterType` type to avoid using `string`, which would enforce the use of the constants you defined here (and prevent using something else by mistake)

Reply:
Currently, we are not using it as func arg anywhere, How about making it `ClusterType` when we have some cases where we are using it as func arg

Reply:
ok, sounds good.